graphy2/core.py

```diff
@@ -75,40 +78,137 @@ def scatter_plot(
-                                                                   self._data["effect"], self._data["Relative Weight"])):
-            plt.plot([lower, upper], [index+0.5, index+0.5])  # draw line
-            plt.plot([effect], [index+0.5], marker="s", markersize=weight*weight_area)  # draw weighted effect
-
-        # dotted line
-        plt.plot([1, 1], [0, len(self._data.index)], "--", color="Black")
-        plt.show()
-
-        # todo These need to be put together in a Data -> Plot format
+        ax2.set(
+            xlim=(min(self._data["lower_bound"]), max(self._data["upper_bound"])),
+            ylim=(0, len(self._data.index)),
+        )
+        # Make sure the plots are ordered correctly against the table
+        plt.gca().invert_yaxis()
+
+        # Altering the appearance of the axis ticks and labels
+        ax2.tick_params(
+            axis="y",  # changes apply to the x-axis
+            which="both",  # both major and minor ticks are affected
+            labelleft=False,  # ticks along the bottom edge are off
+            top=False,
+        )  # ticks along the top edge are off
+        sns.despine(left=True, bottom=False, right=True)
+        ax2.xaxis.set_minor_locator(
+            AutoMinorLocator()
+        )  # This doesn't seem to do anything right now?
+
+        # Drawing the actual lines and squares on the plot
+        for index, (lower, upper, effect, weight) in enumerate(
+            zip(
+                self._data["lower_bound"],
+                self._data["upper_bound"],
+                self._data["effect"],
+                self._data["Relative Weight"],
+            )
+        ):
+            ax2.plot([lower, upper], [index + 0.5, index + 0.5])  # draw line
+            ax2.plot(
+                [effect], [index + 0.5], marker="s", markersize=weight * weight_area
+            )  # draw weighted effect
+        # Draw the dotted line
+        ax2.plot([1, 1], [0, len(self._data.index)], "--", color="Black")
+
+        # Table Plot
+        ax1.axis("off")  # Turn off the axis labels
+        # Set up the table
+        tbl = ax1.table(
+            cellText=df_reduced.values,
+            colLabels=[
+                "$\\bf{Study}$",
+                "$\\bf{Case\\ (n/N)}$",
+                "$\\bf{Control\\ (n/N)}$",
+                "$\\bf{OR\\ (95\\%\\ CI)}$",
+                "$\\bf{Weight}$",
+            ],
+            edges="open",
+            loc="center",
+            cellLoc="center",
+        )
 
-    def image_stack_figure(self, dimensions=None, z_list=None, elevation=30, rotation=45, down_sampling=6,
-                           vertical_stack=True):
+        # These lines set the row height so that they fit the plot.
+        # This ensures each row is sized to line up with the plot on the right.
+        table_props = tbl.properties()
+        table_cells = table_props["child_artists"]
```

Comment:
This is the core line that didn't work for me and was changed to this instead

```suggestion
         for cell in tbl.properties()["children"]:
            cell.set_height(1 / no_of_rows)
```